examples/webhooks/webhooks_test.go

```diff
@@ -1,100 +1,67 @@
 package webhooks
 
 import (
-	"bytes"
-	"encoding/json"
 	"fmt"
+	"log"
 	"net/http"
-	"net/http/httptest"
-	"testing"
-	"time"
-
-	"github.com/google/uuid"
-	"github.com/stretchr/testify/require"
 
 	"github.com/moovfinancial/moov-go/pkg/mhooks"
-	"github.com/moovfinancial/moov-go/pkg/moov"
 )
 
-func TestParseEvent(t *testing.T) {
-	var (
-		timestamp = "2024-04-26T21:20:55Z"
-		secret    = "my-webhook-signing-secret"
-		signature = "6231d03752de6963087e6aea1c78a27a0617b6df1c071195f30ed85defe34e02fd0bf3995949fe12dafd747c42de9cfae03b8aafcf69cceba5495f4c7b719d82"
-
-		accountCreated = mhooks.AccountCreated{
-			AccountID: uuid.NewString(),
+// Example handler func for processing a single event type (AccountCreated)
```

Comment:
If you want to see a cleaner diff, i would recommend just reading the file: https://github.com/moovfinancial/moov-go/blob/bbbf66f72b8d083bd90fa79150403483a98a3023/examples/webhooks/webhooks_test.go